application/batch/get_novel_data.py

```diff
@@ -153,3 +160,51 @@ def get_insert_novel_keyword_list(insert_data_list: list[NarouData]):
             novel_keyword_data["keyword"] = keyword
             insert_novel_keyword_list.append(novel_keyword_data)
     return insert_novel_keyword_list
+
+
+def generate_flow_run_name():
+    """実行時の名前を生成."""
+    date = jst_now().strftime("%Y%m%d")
+    return f"get_novel_data_{date}"
+
+
+def get_chunk_list(input_list: list, chunk_size: int):
+    """指定されたサイズでリストを分割する関数.
+
+    Parameters:
+        input_list (list): 分割するリスト.
+        chunk_size (int): 各チャンクのサイズ .
+
+    Returns:
+        list: 分割されたリストのリスト。
+    """
+    return [
+        input_list[i : i + chunk_size]
+        for i in range(0, len(input_list), chunk_size)
+    ]
+
+
+@flow(flow_run_name=generate_flow_run_name)
+def get_novel_data():
+    """小説情報取得バッチ."""
+    # 1-1 get_target_from_dbを実行
+    # 1回だけ実行
+    # data_listが空の場合は終了
+    console_logger.info("flow-1-1 get_target_from_dbを実行")
+    data_list = get_target_from_db()
+    if not data_list:
+        console_logger.info("対象のデータが存在しないため、処理を終了")
+        return
+
+    # 1-2 fetch_novel_infoを実行
+    # 500件ずつ区切って並列実行
+    # (APIの最大取得件数が500件なため)
+    console_logger.info("flow-1-2 fetch_novel_infoを実行")
+    chunke_data_list = get_chunk_list(data_list, 500)
+    fetch_data_list = fetch_novel_info.map(chunke_data_list).result()
```

Comment:
```suggestion
    chunk_data_list = get_chunk_list(data_list, 500)
    fetch_data_list = fetch_novel_info.map(chunk_data_list).result()
```

typoしているので、修正お願いします 🙏 
多分 `commit suggestion` でレビューした内容をそのままコミットできるので、サクッと適用してもらえると嬉しいです 👍